File: 04 - Mad Lib/Haskell/main.hs
main :: IO()

main = do
    putStrLn "What is the verb? "
    verb <- getLine
    putStrLn "What is the adjective? "
    adj <- getLine
    putStrLn "What is the noun? "
    noun <- getLine
    putStrLn "What is the adverb? "
    adv <- getLine
    putStrLn("Do you " ++ verb ++ " your " ++ adj ++ " " ++ noun ++ " " ++ adv ++ "? That's hilarious!")

File: 04 - Mad Lib/C++/main.cpp
#include <iostream>

using namespace std;

string stdinput(string phrase){
    string input;
    cout << phrase;
    getline(cin, input);
    return input;
}

int main(void)
{
    string noun, verb, adjective, adverb;

    noun = stdinput("Enter a noun: ");
    verb = stdinput("Enter a verb: ");
    adjective = stdinput("Enter a adjective: ");
    adverb = stdinput("Enter a adverb: ");

    cout << "Do you " << verb << " your " << adjective << " " << noun << " quickly? That's hilarious!" << endl;

    return 0;
}
